MJ68-sleep.ict

```diff
@@ -468,7 +644,69 @@ types:
     extends:
       ctype: Bug
     uid: 2d208313-f3b8-4520-9b57-897ebb99c198
-    lastmod: 1606608303676
+    lastmod: 1606600496561
+  - name: clock_desk_1
+    depth: 9
+    oncreate: |-
+      this.currentFace = 1;
+
+      // For some reason, copy can't access Room OnCreate properties right away
+      var initTimer = ct.timer.add(500, 'roomInit');
+
+      initTimer.then(() => {
+          // 
+          this.timeStep = (ct.room.timerLength-500)/6;
+
+          this.interval = setInterval(() => {
+              if (this.currentFace < 7) {
+                  this.currentFace++;
+                  this.tex = 'clock_desk_' + this.currentFace;
+              }
+              else {
+                  clearInterval(this.interval);
+              }
+          }, this.timeStep);
+
+      });
+    onstep: >-
+      // Change clock face at every 6th interval so it doesn't matter what the
+      starting time is
+
+
+
+
+
+      /*this.clockTimer -= ct.delta;
+
+      console.log(this.clockTimer)
+
+      if (this.clockTimer <= 0) {
+          // Reset timer
+          this.clockTimer = this.timeStep;
+
+          // Change face texture if not the last one
+          if (this.currentFace < 7) {
+              this.currentFace += 1;
+              this.tex = 'clock_desk_' + this.currentFace;
+          }
+          console.log(this.currentFace);
+      }*/
```

Comment:
Committing commented out blocks usually isn't great. Easy to get lost once it's no longer in a diff. Adding a searchable TODO comment explaining what needs to be accomplished here would be helpful.